pydoctor/epydoc2stan.py

```diff
@@ -20,11 +20,6 @@
 from pydoctor.epydoc.markup import DocstringLinker, ParsedDocstring
 import pydoctor.epydoc.markup.plaintext
 
-try:
-    import exceptions
-except ImportError:
-    exceptions = builtins
```

Comment:
We should probably remove this cross-version compat shims, and just use `builtins` explicitly below.

Reply:
The problem with using `builtins` is that since Python 3 it is no longer possible to tell which names represent exceptions and which represent non-exception types based on the module they're defined in; that's exactly what was causing the bug. The fix instead uses `issubclass()` to check whether a name belongs to an exception class.